src/config/dynamic-theme-fixes.config

```diff
@@ -3505,6 +3505,15 @@ CSS
 
 ================================
 
+matters.news
+
+CSS
+.jsx-178004874.container {
```

Comment:
`.jsx-178004874.`
JSX seams to be [React JSX](https://reactjs.org/docs/introducing-jsx.html),
and `178004874` seems to be some randomized number.

So I guess a 
```CSS
section[class$="container"][style^="background-image"]
```
Also isn't a INVERT better?

Reply:
Thanks! I'll test it later.

Reply:
INVERT inverts its child elements.

```
INVERT
section[class$="container"][style^="background-image"]
section[class$="container"][style^="background-image"] > h2
section[class$="container"][style^="background-image"] > p
section[class$="container"][style^="background-image"] > div
```

![sshot-009](https://user-images.githubusercontent.com/36977733/91626673-c9543680-e9a0-11ea-8146-4e2015cfe49a.png)

or

```
CSS
section[class$="container"][style^="background-image"] {
    background-image: none !important;
}
```

![sshot-008](https://user-images.githubusercontent.com/36977733/91626674-cd805400-e9a0-11ea-86bf-3b8dad530fdf.png)

which is better?

Reply:
Second one

Reply:
Updated, thanks!